lib/image-type.ts

```diff
@@ -15,6 +15,15 @@ export function getImageType(targetImage: string): ImageType {
   }
 }
 
+export function getImageIdentifier(target: string): string | undefined {
+  const targetSplit = target.split(":");
+  if (targetSplit[0] === "docker-archive" || targetSplit[0] === "oci-archive") {
```

Comment:
might be good to have some comments, for example here, to clarify why we are looking for these strings and excluding them (I think in this case it is because of requests coming from the CLI? so need knowledge of this if there is no comment), wdyt?

Reply:
Yeah agreed comments would be good - I'll add some before I un-draft.